backend/molgenis-emx2-graphql/src/test/java/org/molgenis/emx2/graphql/TestGraphqlCrossSchemaRefs.java

```diff
@@ -58,7 +58,7 @@ public void test() throws IOException {
   void testCrossSchemaTablesAreInSchemaEndpoint() throws IOException {
     String result =
         execute("{_schema{tables{name,id,schemaName,schemaId}}}").at("/_schema").toString();
-    assertTrue(result.contains(schemaName1));
+    // we stop with cross schema inclusion here assertTrue(result.contains(schemaName1));
```

Comment:
what is the impact of disabling this?

Reply:
before javascript client this was needed. I actually thought this was already gone. So again, should not be problematic but we should test the data explorer and schema editor for a cross schema model (e.g. data-catalogue)